tests/system/Helpers/FormHelperTest.php

```diff
@@ -943,7 +943,7 @@ public function testValidationShowError()
 
         $html = validation_show_error('id');
 
-        $this->assertSame('<span class="help-block">The ID field is required.</span>' . "\n", $html);
+        $this->assertSame('<span class="help-block">The ID field is required.</span>' . PHP_EOL, $html);
```

Comment:
Why `PHP_EOL`?

Reply:
Passed for PHPCSFixer, i forgot template location render for this.

```html
<span class="help-block">The ID field is required.</span>

```

Reply:
![image](https://user-images.githubusercontent.com/97607754/196897485-3e106d3d-440b-40f4-911f-1f88eca3b7ef.png)

See in https://github.com/codeigniter4/CodeIgniter4/blob/4.3/system/Validation/Views/single.php#L1, have 2 line

Reply:
What's is it? I don't get what you want to say.

Reply:
![image](https://user-images.githubusercontent.com/97607754/197774007-217f83ed-a844-45cb-be09-6df7ee627b23.png)
@paulbalandan Still same

Reply:
Check the end of line code of `system/Validation/Views/single.php`.
All the source code should use LF.

Reply:
Going to `system/Validation/Views/single.php`, click the `CRLF` status on bottom right. The command palette should open. Click on `LF`. Then save. If Git recognizes the change, it means you might have committed the CRLF EOL sequence for your fork.

To fix:
1. Set Git to use LF on windows
```
git config --global core.autocrlf input
git config --global core.eol lf
```

2. Delete all files in your local machine, except for the `.git` folder. Then, re-pull from origin.